### DISCUSSION TITLE
Transfer trained model use cases

### DISCUSSION THREAD

**Discussion by mohammadbashiri** [sinzlab/nnfabrik#107] [General]
What are common use cases of this new trained model?

**Comment by mohammadbashiri** [comment]
I think in general usually some part of the model is transferred. Here are some use-cases I could think of:
1) stage 0: train model -> stage 1: use trained core in previous stage
2) stage 0: train model -> stage 1: continue training (some part of) the model with a different trainer
3) stage 0: train model -> stage 1: continue training (some part of) the model with a different dataset
4) what else?

**Reply by KonstantinWilleke** [reply]
I think that (1) can be extended to readout and parts of the readout, too. So positions/features only, for example.

**Reply by chbehrens** [reply]
Our usecase is stage 0: train full on a large dataset, stage 1: train only a new readout on a new smaller dataset during closed loop experiments. Another possible usecase would be to train one model on a certain dataset, then take the core, add another layer or two and train use it to train on data recorded further upstream.
